.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
```

Comment:
Can we link to this: https://github.com/psf/black/issues/315 , since it explains "how" the 2 conflict?  (or maybe a lengthy reason for the exclusion)

Reply:
I guess, I tried not to change the previous contents too much, leaving that for follow-ups.

Reply:
NP, can we log an issue to have it updated in a follow up PR so it's not forgotten?

Reply:
https://github.com/ansible/ansible-navigator/issues/721